lib/Minify/Minify.php

```diff
@@ -388,7 +388,7 @@ public static function serve($controller, $options = array())
 				self::$lastServed['fullCacheId'] = $fullCacheId;
 				self::$_cache->display($fullCacheId);
 			} else {
-				echo $content['content'];
+				echo esc_html( $content['content'] );
```

Comment:
Check if the content should get escaped.

Reply:
Switched to wp_kses with call to Util_Ui::get_allowed_html_for_wp_kses_from_content for allowed HTML tags